src/main/java/com/hubspot/jinjava/util/ScopeMap.java

```diff
@@ -2,13 +2,19 @@
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class ScopeMap<K, V> implements Map<K, V> {
+  private static final Logger LOG = LoggerFactory.getLogger(ScopeMap.class);
```

Comment:
I see we use `ENGINE_LOG` everywhere else in jinjava, should we keep using that?

Reply:
Good call